MdeModulePkg/Include/Protocol/PciDoe.h

```diff
@@ -0,0 +1,73 @@
+/** @file
+  EFI PCI I/O Protocol provides the basic Memory, I/O, PCI configuration,
+  and DMA interfaces that a driver uses to access its PCI controller.
+
+  Copyright (c) 2006 - 2018, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PCI_PROTOCOL_DOE_H_
+#define PCI_PROTOCOL_DOE_H_
+
+typedef struct _EFI_PCI_IO_DOE EFI_PCI_IO_DOE;

```

Comment:
For the protocol defined in MdeModulePkg, the prefix should be `EDKII_`, not `EFI_`. (It is historic reason that some protocols are still using `EFI_`. The new one should always use `EDKII_`